lib/ansible/modules/dnf.py

```diff
@@ -36,7 +36,8 @@
 
   list:
     description:
-      - Various (non-idempotent) commands for usage with C(/usr/bin/ansible) and I(not) playbooks. See examples.
+      - Various (non-idempotent) commands for usage with C(/usr/bin/ansible) and I(not) playbooks.
+        Use M(ansible.builtin.package_facts) instead.
```

Comment:
It's a bit of a gray area. It hasn't officially been deprecated afaik, but we don't want people using it. See https://github.com/ansible/ansible/issues/78069 for details. We could change the text here to 'we recommend Using foo instead."

Reply:
Oh right. I don't know why I didn't see the issue that was obviously linked in this PR description before. I've got the context now though.

I'm going to update my suggested change.

Reply:
@samccann OK I made the update. I don't know if we want to say exactly that but I feel like there should be a reason to use foo instead of bar. That makes it more compelling imho.